### PR TITLE
Benchmark system

### PR DESCRIPTION
**Pull request by RaphaelKreft**
- Added Shell script to be able to run benchmarks quickly and many in sequence without need for manual restart
- Added a common benchamrk conf that is meant to be run with all our model configs to have comparable results.

Please especially look at the common config whether these params make sense for you.

### CODE REVIEW

File: gluefactory/configs/COMMON_BENCHMARK_CONF.yaml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+model:  # TODO: REPLACE MODEL WITH YOU MODEL CFG YOU WANT TO BENCHMARK
+  name: two_view_pipeline
+  extractor:
+    name: extractors.joint_point_line_extractor
+    aliked_model_name: "aliked-n16"
+    max_num_keypoints: 2048  # setting for training, for eval: -1
```
Comment (hardik01shah) [inline gluefactory/configs/COMMON_BENCHMARK_CONF.yaml:6]: You mentioned a threshold on the score map instead of setting max_num_points right?

**Review by hardik01shah — COMMENTED** [review]
Looks good apart from the minor comment about fixing max_num_keypoints instead of a detection threshold!

File: gluefactory/configs/COMMON_BENCHMARK_CONF.yaml (hunk 1 above)
Comment (RaphaelKreft) [reply gluefactory/configs/COMMON_BENCHMARK_CONF.yaml:6]: yes, in theory we should set a threshold that returns enough good keypoints. As we don't have this yet and benchmarks are set to use a max num of kp, It should be fine for now to use ex the top 2048 keypoints

Comment (RaphaelKreft) [reply gluefactory/configs/COMMON_BENCHMARK_CONF.yaml:6]: I will open issue regarding fixing threshold later so we remember doing that but merge for now

**Pull request merged**